dashboard/modules/job/common.py

```diff
@@ -85,6 +90,11 @@ def get_status(self, job_id: str) -> Optional[JobStatusInfo]:
         else:
             return pickle.loads(pickled_status)
 
+    def get_all_jobs(self) -> Dict[str, JobStatusInfo]:
```

Comment:
@architkulkarni @tchordia this is also a step towards being able to support listing jobs: we just need to add some more metadata and call this interface.